tests/test_unittest/test_gtscript_frontend.py

```diff
@@ -139,7 +139,7 @@ def definition_func(inout_field: gtscript.Field[float]):
             with computation(PARALLEL), interval(...):
                 inout_field = inout_field[0, 0, 0] + WRONG_VALUE_CONSTANT
 
-        with pytest.raises(gt_frontend.GTScriptSymbolError, match=r".*WRONG_VALUE_CONSTANT.*"):
+        with pytest.raises(gt_frontend.GTScriptDefinitionError, match=r".*WRONG_VALUE_CONSTANT.*"):
```

Comment:
We now catch the error sooner than previously, so the type of exception changed.

Reply:
Perhaps that error should be switched to `SymbolError` instead of `DefinitionError`.